src/containers/itemCollection/itemCollection.jsx

```diff
@@ -51,15 +51,58 @@ export class ItemCollection extends Component {
         })
     }
 
+    // This function splits a string by tabs/newlines and individually
+    // submits each item and then adds another edit item.
+    handleEditItemPaste = (pasteString) => {
+        //TODO does this work on Windows? Does it need to check for carriage return?
+        var splitStrings = pasteString.split(/[\t\n]/)
+
+        for (let itemName of splitStrings) {
+            //TODO Need to double check whether this logic of interacting with other functions is correct
+            const item = {
+                _id: uuid('item'),
+                name: itemName,
+                size: 1
+            }
+    
+            // Prevent the addition of an empty item, null item, or all whitespace item
+            if (item.name === null || item.name.match(/^\s*$/) !== null) {
```

Comment:
Move this out into a common function. That way we can build out a common, name verification utility that also takes into account if the name has been duplicated.

<img width="652" alt="Screenshot 2019-03-10 08 35 09" src="https://user-images.githubusercontent.com/461874/54087399-c3334b80-430f-11e9-9f3f-00b7d7872d1b.png">